src/app/view/perspectives/calendar/calendar-perspective.component.html

```diff
@@ -27,6 +27,7 @@
                      [permissions]="permissions$ | async"
                      [constraintData]="constraintData$ | async"
                      [query]="data.query"
+                     [toolbarOpened]="sidebarOpened$.asObservable()"
```

Comment:
Passing observable as property is not good practise. Why don't you just put async pipe here?